dev-postgres-powersync/docker-compose.yml

```diff
@@ -55,7 +58,11 @@ services:
     restart: unless-stopped
 
 volumes:
-  postgres-data:
+  postgres-dev-data:
+
+networks:
+  default:
+    name: wger-dev-network
```

Comment:
why are we re-introducing the network override? didn't we just remove this? and what does it have to do with pg_ivm?

Reply:
it has nothing to do with it, but I wanted to keep the dev services separate, otherwise it's not possible to run both the production and the dev stack at the same time since they would both access the same db, etc.